Sources/Tagged/Tagged.swift

```diff
@@ -1,3 +1,5 @@
+import Foundation
```

Comment:
import is needed to make `localizedDescription` available.

Do you guys support linux? Do we need `#if canImport(Foundation)` or something? I have no experience with that...